tests/io_test.py

```diff
@@ -596,6 +596,15 @@ def testSmallStringArrayToHDF5(self):
             b2 = ak.load(f"{tmp_dirname}/single_string", dataset="b1")
             self.assertEqual(str(b1), str(b2))
 
+    def testUint64ToHDF5(self):
+        """
+        Test conversion to and from numpy array / pdarray using unsigned 64bit integer (uint64)
+        """
+        npa1 = np.array([18446744073709551500, 18446744073709551501, 18446744073709551502])
```

Comment:
Do we know this is uint64 just based on the number of digits?

Reply:
18446744073709551615 is uint64 max value I think (based on https://docs.microsoft.com/en-us/dotnet/api/system.uint64.maxvalue?view=net-6.0)  although I can add a specific dtype on this to make sure.